frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -36,11 +36,24 @@ function FileSearcher:registerKeyEvents()
 end
 
 function FileSearcher:onShowFileSearch(search_string)
-    local search_dialog
-    local check_button_case, check_button_subfolders, check_button_metadata
+    local search_dialog, check_button_case, check_button_subfolders, check_button_metadata
+    local function _doSearch()
+        local search_str = search_dialog:getInputText()
+        if search_str == "" then return end
+        self.search_string = search_str
+        G_reader_settings:saveSetting("file_search_string", search_str)
```

Comment:
In ReaderSearch, it's called `fulltext_search_last_search_text`.
Would be nice if these read similar :) ie. `file_search_last_search_text`